elyra/pipeline/airflow/processor_airflow.py

```diff
@@ -597,6 +598,12 @@ def render_volumes(self, elyra_parameters: Dict[str, ElyraProperty]) -> str:
         for v in elyra_parameters.get(pipeline_constants.MOUNTED_VOLUMES, []):
             str_to_render += f"""
                 Volume(name="{v.pvc_name}", configs={{"persistentVolumeClaim": {{"claimName": "{v.pvc_name}"}}}}),"""
+        # set custom shared memory size
+        shm = elyra_parameters.get(pipeline_constants.KUBERNETES_SHARED_MEM_SIZE)
+        if shm is not None and shm.size:
+            config = f"""configs={{"emptyDir": {{"medium": "Memory", "sizeLimit": "{shm.size}{shm.units}"}}}}"""
```

Comment:
`shm.size` and `shm.units` doesn't look like they are defined in the `ElyraProperty` ABC.

Reply:
These come from the object-valued property subclass: `CustomSharedMemorySize`.